packages/entity-browser/src/modules/detailView/sagas.js

```diff
@@ -43,6 +47,22 @@ export function* loadEntity({payload}) {
   yield put(actions.setEntity(entity))
 
   const formValues = yield call(entityToFormValues, entity)
+
+  // initialize store for fields of the types `entity` and `entity-list`
+  const keys = Object.keys(entity.paths)
```

Comment:
Refactoring suggestion: 
- Create Util method "getRelatedEntitesStoreInitialValues" (or something like that) that returns an object with this structure:
 { key: fieldstore, key2: fieldstrore}
- In saga iterate over key and call put


This way nice2 specific logic is outside of sages and not mixed